### PR TITLE
networkd: bridge add support to configure multicast_to_unicast

### LINKED ISSUE #10649: [networkd] Support for additional bridge link properties
**systemd 239 on Arch**
I'm running a bridge that includes 2 wireless/hostapd links. I'm currently enabling *hairpin_mode* for these wireless links as shown below and would like to be able to set *multicast_to_unicast* on each of these bridge ports in the same way.

```
[Match]
Name=wifi*

[Network]
Bridge=br0

[Bridge]
HairPin=true
```
While my specific use case is for *multicast_to_unicast* support, there are additional bridge port properties available that can't currently be set via networkd.

My solution at the moment is just to write directly to the sysfs property when the bridge link appears using a separate systemd path/service.

### PR DESCRIPTION
**Pull request by ssahani**
closes #10649

### CODE REVIEW

File: src/basic/missing.h [hunk 1]
```diff
@@ -984,7 +984,30 @@ struct input_mask {
...
+#define IFLA_BRPORT_MESSAGE_AGE_TIMER 21
+#define IFLA_BRPORT_FORWARD_DELAY_TIMER 22
+#define IFLA_BRPORT_HOLD_TIMER 23
+#define IFLA_BRPORT_FLUSH 24
+#define IFLA_BRPORT_MULTICAST_ROUTER 25
+#define IFLA_BRPORT_PAD 26
+#define IFLA_BRPORT_MCAST_FLOOD 27
+#define IFLA_BRPORT_MCAST_TO_UCAST 28
+#define IFLA_BRPORT_VLAN_TUNNEL 29
+#define IFLA_BRPORT_BCAST_FLOOD2 30
```
Comment (yuwata) [inline src/basic/missing.h]: Typo: IFLA_BRPORT_BCAST_FLOOD

**Comment by ssahani** [comment]
@evverx anything I missed please take a look "fuzzer"

**Comment by evverx** [comment]
@ssahani The script is complaining that https://github.com/systemd/systemd/blob/master/test/fuzz/fuzz-network-parser/directives.network hasn't been updated. Could you add `MulticastToUnicast=` to the "Bridge" section there?

File: man/systemd.network.xml [hunk 2]
```diff
@@ -1721,6 +1721,17 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>MulticastToUnicast=</varname></term>
+          <listitem>
+            <para>Takes a boolean. Controls whether the bridge multicast to unicast works on top
+            of the multicast snooping feature of the bridge. Which means unicast copies are only
```
Comment (poettering) [inline man/systemd.network.xml]: hmm, this needs better wording. "the bridge multicast to unicast" is something that doesn't make sense to me. @keszybz any suggestion how to make this better?

**Comment by poettering** [comment]
(also needs rebase)

**Comment by ssahani** [comment]
Thanks @poettering  for review . PTAL.

**Comment by yuwata** [comment]
Adding test case is welcome :-)

**Comment by yuwata** [comment]
LGTM. The documentation seems to be good, at least better than the previous. I will test this later.

**Pull request merged**